src/backport.ts

```diff
@@ -152,18 +152,20 @@ const getFailedBackportCommentBody = ({
     "```",
     "To backport manually, run these commands in your terminal:",
     "```bash",
+    "upstream_name=qgis",
```

Comment:
Is this still used?

Reply:
now it is ;-)

Reply:
It is general purpose

Reply:
Thanks a lot! I just tried it and it just works! :1st_place_medal: 

However, there is something I don't understand... I can't figure out what `qgis` is here.
I looked at your profile and found that you work with the QGIS framework, so I thought this line was something you needed specifically for yourself (that's why I had asked you if this was general purpose or not).
Then, assuming this is a general purpose solution, the value of `upstream_name` could be anything else and it wouldn't make a difference?

Reply:
Great!

In case a backport fails, these are the instructions given in a comment to backport manually. `qgis` would be a name for a remote but I'm unsure about the best / most generic / most understandable way to phrase this, hence my silence...